src/hooks/useComics/useComics.js

```diff
@@ -0,0 +1,34 @@
+import {useEffect, useState} from 'react';
+import axios from 'axios';
+
+const URL =
+  'https://gateway.marvel.com/v1/public/comics?ts=1&apikey=64db28cd226245f53f4c8ab2c62bbafd&hash=07ed8cc60a3326b2ced80fe0b8056835';
```

Comment:
Bunlar için `react-native-config` paketini kullanalım. URL .env'de saklanırsa daha sağlıklı olur.